### PR TITLE
update chat2query api, add doc for data_summary_id

### PR DESCRIPTION
**Pull request by joyjoyhong**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/use-chat2query-api.md [hunk 1]
```diff
@@ -255,6 +255,7 @@ In the preceding code, the request body is a JSON object with the following prop
```
Comment (Oreoxmt) [inline tidb-cloud/use-chat2query-api.md:255]: Please update line 254 to:

```
The request body is a JSON object with the following properties:
```

File: tidb-cloud/use-chat2query-api.md [hunk 2]
```diff
@@ -255,6 +255,7 @@ In the preceding code, the request body is a JSON object with the following prop
 
 - `cluster_id`: _string_. A unique identifier of the TiDB cluster.
 - `database`: _string_. The name of the database.
+- `data_summary_id`: _integer_. The ID of the data summary you want to use for generate SQL. If you do not provide this parameter, the API will use the default data summary. Only works if `cluster_id` and `database` not provided.
```
Comment (Oreoxmt) [inline tidb-cloud/use-chat2query-api.md:258]:
```suggestion
- `data_summary_id`: _integer_. The ID of the data summary used to generate SQL. This property only takes effect if `cluster_id` and `database` are not provided. If you do not specify `cluster_id`, `database`, and this property, the API uses the default data summary.
```
Are there any existing documents describing the default data summary?

**Review by Oreoxmt — APPROVED** [review]

File: tidb-cloud/use-chat2query-api.md [hunk 3]
```diff
@@ -251,10 +251,11 @@ curl --digest --user ${PUBLIC_KEY}:${PRIVATE_KEY} --request POST 'https://<regio
 }'
 ```
 
-In the preceding code, the request body is a JSON object with the following properties:
+The request body is a JSON object with the following properties:
 
 - `cluster_id`: _string_. A unique identifier of the TiDB cluster.
 - `database`: _string_. The name of the database.
+- `data_summary_id`: _integer_. The ID of the data summary used to generate SQL. This property only takes effect if `cluster_id` and `database` are not provided. If you do not specify `cluster_id`, `database`, and this property, the API uses the default data summary.
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:258]: What's the default data summary? Is it generated by Data Service for all databases automatically?

Comment (Oreoxmt) [inline tidb-cloud/use-chat2query-api.md:258]:
```suggestion
- `data_summary_id`: _integer_. The ID of the data summary used to generate SQL. This property only takes effect if `cluster_id` and `database` are not provided. If you specify both `cluster_id` and `database`, the API uses the default data summary of the database.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**